portfolio/src/main/webapp/comments.js

```diff
@@ -0,0 +1,34 @@
+/*
+* Purpose: recieves HTTP promise response from CommentsServelt.java, places names and comments into appropriate html container 
+*/
+
+/* fetches from CommentsServlet.java */
+function getComment(){
+    const responsePromise = fetch('/comments');
+    // when server request complete, pass response into handleResponse
+    responsePromise.then(handleResponse);
+}
+
+/*Takes JSON string, converts into JavaScript object*/
+function handleResponse(response){
+    // receives Java Object (ArrayList reponse)
+    const commentListPromise = response.json();
+    commentListPromise.then(addCommentsToDom);
+}
+
+/* Adds names and comments to html page, displaying name: comment */
+function addCommentsToDom(commentList){
```

Comment:
Tiny formatting thing, I think the formatting guide says spaces between braces, so like (commentList) {

Reply:
Good catch Leah!